the-new-developers/gatehopper
# Adding the level countdown functionality

## Changes:
Created `levelCountdown.js` as a scene under the `src/scenes/`. The file contains a countdown functionality that will be used before a level appears to user in **Challenge Mode** or **Practice Mode**.
Also, added the scene to `src/index.js` and referred to it in `index.html`

The following video shows a preview of the countdown:

https://user-images.githubusercontent.com/43936284/127812836-c13e7f7b-2d64-41fc-8709-4169e7acec2e.mp4

Closes #30

***IMPORTANT NOTES***🛑:
1. The function uses fixed numbers for height and width for the scene as the main scene isn't dynamic yet.
2. There is no specific timing for this function to run. The dev who's working on creating levels in **Practice Mode** and **Challenge Mode** can decide if they want to use the code of this function as is, if they want to modify this function to make it accept a parameter, or some other way they have in mind.


## How to test this PR:
Change the active scene in `index.js` to be "**Level Countdown**", and then run `index.html` and the countdown should appear.

## Checklist:
